student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission, name)
+    super(age, parent_permission, name)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please make sure to pass the default values in the initializer as requested in the project requirements. `initialize(age, classroom, parent_permission = true, name = "Unknown")`